daprdocs/content/en/reference/components-reference/supported-bindings/cfqueues.md

```diff
@@ -0,0 +1,236 @@
+- **`cfAPIToken`**: API token with permission to create and edit Workers. You can create it from the ["API Tokens" page](https://dash.cloudflare.com/profile/api-tokens) in the "My Profile" section in the Cloudflare dashboard. Click on "Create token", select the **"Edit Cloudflare Workers"** template, then follow the on-screen instructions to generate a new API token.
+
+When Dapr is configured to manage the Worker for you, when a Dapr Runtime is started it checks that the Worker exists and it's up-to-date. If the Worker doesn't exist, or if it's using an outdated version, Dapr will create or upgrade it for you automatically.
+
+{{% /codetab %}}
+
+{{% codetab %}}
+<!-- Manually provision the Worker script -->
+
+If you'd rather not give Dapr permissions to deploy Worker scripts for you, you can manually provision a Worker for Dapr to use. Note that if you have multiple Dapr components that interact with Cloudflare services via a Worker, you will need to create a separate Worker for each one of them.
+
+To manually provision a Worker script, you will need to have Node.js installed on your local machine.
+
+1. Create a new folder where you'll place the source code of the Worker, for example: `daprworker`.
+2. If you haven't already, authenticate with Wrangler (the Cloudflare Workers CLI) using: `npx wrangler login`.
+3. Inside the newly-created folder, create a new `wrangler.toml` file with the contents below, filling in the missing information as appropriate:  
+  
+  ```toml
+  # Name of your Worker, for example "mydaprqueue"
+  name = ""
+
+  # Do not change these options
+  main = "worker.js"
+  compatibility_date = "2022-12-09"
+  usage_model = "bundled"
+
+  [vars]
+  # Set this to the **public** part of the Ed25519 key, PEM-encoded (with newlines replaced with `\n`).
+  # Example:
+  # PUBLIC_KEY = "-----BEGIN PUBLIC KEY-----\nMCowB...=\n-----END PUBLIC KEY-----
+  PUBLIC_KEY = ""
+  # Set this to the name of your Worker (same as the value of the "name" property above), for example "mydaprqueue".
+  TOKEN_AUDIENCE = ""
+
+  # Set the next two values to the name of your Queue, for example "myqueue".
+  # Note that they will both be set to the same value.
+  [[queues.producers]]
+  queue = ""
+  binding = ""
+  ```
+  
+  > Note: see the next section for how to generate an Ed25519 key pair. Make sure you use the **public** part of the key when deploying a Worker!
+
+4. Copy the (pre-compiled and minified) code of the Worker in the `worker.js` file. You can do that with this command:  
+  
+  ```sh
+  # Set this to the version of Dapr that you're using
+  DAPR_VERSION="release-{{% dapr-latest-version short="true" %}}"
+  curl -LfO "https://raw.githubusercontent.com/dapr/components-contrib/${DAPR_VERSION}/internal/component/cloudflare/workers/code/worker.js"
```

Comment:
Reviewers: this command won't work until we cut the first release in the 1.10 branch. In the meanwhile, if you need to test this, use:

```sh
curl -LfO "https://raw.githubusercontent.com/dapr/components-contrib/master/internal/component/cloudflare/workers/code/worker.js"
```